website/conferences/utils.py

```diff
@@ -56,7 +56,9 @@ def prepare_contributors(admins):
 
 def upload_attachment(user, node, attachment):
     attachment.seek(0)
-    name = '/' + (attachment.filename or settings.MISSING_FILE_NAME)
+    name = (attachment.filename or settings.MISSING_FILE_NAME)
+    if name[0] != '/':
+        name = '/' + name
```

Comment:
There's a python method to do this. I can't remember which, and I have a meeting, but I found it via @laurenrevere so she can probably tell you what it is.

Reply:
`urllib.parse.urljoin`

Reply:
In python 2.7 it's `urlparse.urljoin`, though.